ee_tests/config/local_osio.conf.sh.template

```diff
@@ -1,4 +1,5 @@
 # Test config for running e2e tests against remote osio
+export DEBUG="false"
```

Comment:
Just wondering - why would we not want to set this to DEBUG for all tests?

Reply:
because you are not debugging all the time?